.github/ISSUE_TEMPLATE/installation-problem.yaml

```diff
@@ -0,0 +1,43 @@
+name: Installation Problem
+description: Report an issue encountered during installation
+title: ''
+labels: [ 'bug' ]
+assignees: []
+
+body:
+- type: markdown
+  attributes:
+    value: |
+      ## Relevant Discussion  
+      Most installation problems are related to dependencies rather than Pyccel itself.  
+      Please link to a discussion in the [Q&A section](https://github.com/pyccel/pyccel/discussions/categories/q-a) where the problem was investigated.
```

Comment:
Should this be a textarea with the discussion? Related to the comments above. Installation issues are usually found in the Q&A section of the discussions:
https://github.com/pyccel/pyccel/discussions/categories/q-a

and should be asked there first to ensure that the problem is actionable